e2e-tests/experiments/functional/lvm-controller-high-availability/run_litmus_test.yml

```diff
@@ -0,0 +1,26 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  generateName: lvm-controller-high-availability-
+  namespace: litmus
+spec:
+  template:
+    metadata:
+      labels:
+        name: lvm-controller-high-availability
+    spec:
+      serviceAccountName: litmus
```

Comment:
@w3aman can we avoid try to use litmus namespace in tetcase.

Reply:
@nsathyaseelan i think it will be good to keep lvm-e2e related objects in any one namespace. we can avoid litmus name. should i use `lvm-e2e` namespace and change crds name as well  ?